src/bot/api.rs

```diff
@@ -1,15 +1,16 @@
 use crate::{
     bot::Bot,
     requests::{
-        ChatId, EditMessageLiveLocation, ForwardMessage, GetFile, GetMe,
-        SendAudio, SendLocation, SendMediaGroup, SendMessage, SendPhoto,
-        StopMessageLiveLocation, AnswerPreCheckoutQuery, AnswerShippingQuery,
+        AnswerPreCheckoutQuery, AnswerShippingQuery, ChatAction, ChatId,
+        EditMessageLiveLocation, ForwardMessage, GetFile, GetMe,
         KickChatMember, PinChatMessage, PromoteChatMember, RestrictChatMember,
-        SendChatAction, SendContact, SendPoll, SendVenue, SendVideoNote,
-        SendVoice, UnbanChatMember, UnpinChatMessage, ChatAction
+        SendAudio, SendChatAction, SendContact, SendLocation, SendMediaGroup,
+        SendMessage, SendPhoto, SendPoll, SendVenue, SendVideoNote, SendVoice,
+        StopMessageLiveLocation, UnbanChatMember, UnpinChatMessage,
     },
-    types::{InputFile, InputMedia, ChatPermissions},
+    types::{ChatPermissions, InputFile, InputMedia},
 };
+use std::borrow::Cow;
```

Comment:
There is an unspoken rule of `use` order: 
1. `std::`
2. uses from other crates 
3. `crate::`

Reply:
OK.